sql/src/main/java/io/crate/planner/consumer/ConsumingPlanner.java

```diff
@@ -55,12 +56,17 @@ public ConsumingPlanner(AnalysisMetaData analysisMetaData) {
     @Nullable
     public Plan plan(AnalyzedRelation rootRelation) {
         ConsumerContext consumerContext = new ConsumerContext(rootRelation);
+        return plan(rootRelation, consumerContext);
```

Comment:
these changes are unrelated are they? Please to them in a separate commit or even better in a PR against master